test_regress/t/t_concat_string.py

```diff
@@ -11,6 +11,8 @@
 
 test.scenarios('simulator')
 
+signal.alarm(2)  # 2s timeout
```

Comment:
Thanks for the pull!  Add "import signal" above to appease lint, then once tests pass feel free to squash merge.